### ISSUE TITLE
FID and IS results

### ISSUE THREAD

**Issue by Algolzw** [valeoai/Halton-MaskGIT#37]
Hi, thanks for the great work! When I test the pretrained model (vit-base, image size 256) using the `eval_test_to_img.sh` script, it shows quite different FID and IS results with "halton" and "confidence" samplers, as shown below:

<img width="700" height="45" alt="Image" src="https://github.com/user-attachments/assets/361c4de2-d188-4771-bf7f-a611f7f00c3a" />
<img width="719" height="48" alt="Image" src="https://github.com/user-attachments/assets/cea14f98-1956-4e95-bc80-1e2e47a68a6a" />

The confidence sampler produces a very high IS score (364!) but low FID (15.9). I just want to check if this is correct? And if not, then what's the correct hyperparameter for the confidence sampler?

**Comment by llvictorll** [comment]
Hello, yes, I think this is correct. Of course, there can be some randomness during training and sampling, but the results look consistent with my experiments. You can always play with the cfg and softmax temperature to control the trade-off between FID and IS

**Comment by Algolzw** [comment]
Thanks!
